.dockerignore

```diff
@@ -0,0 +1,7 @@
+.gitignore
+.env
+node_modules
+README.md
+STYLE-GUIDE.md
+npm-debug.log
+client
```

Comment:
I see you dockerignored your client here. I did that originally but was having issues when i was trying to live edit from within a container. (maybe it was because I was not running webpack, but that doesn't touch the bundle anyways) But wouldn't ignoring your client not register your changes, in your container?

Reply:
I haven't tried live editing. Thanks for letting me know!